counter.go

```diff
@@ -33,3 +33,9 @@ func (c *DefaultCounter) Get() int64 {
 func (c *DefaultCounter) Set(val int64) {
 	atomic.StoreInt64(&c.val, val)
 }
+
+// AddAndGet provides thread safe way to increase counter
```

Comment:
@LopatkinEvgeniy, let's simplify the comment, e.g., 
`AndGet add val to a counter and returns an updated value.`